test/functional/test_framework/script.py

```diff
@@ -385,6 +385,23 @@ def encode(obj):
             r[-1] |= 0x80
         return bytes([len(r)]) + r
 
+    @staticmethod
+    def decode(vch):
+        # We assume valid push_size and minimal encoding
+        value = vch[1:]
+        # Mask for all but the highest result bit
+        num_mask = (2**(len(value)*8) - 1) >> 1
+        if len(value) == 0:
```

Comment:
If @promag:s suggestions is not implemented: Use `if not value:`

Reply:
fwiw I wouldn't know hot to read that suggestion properly, sorry